### ISSUE TITLE
add other wikimedia properties

### ISSUE THREAD

**Issue by edsu** [edsu/wikistream#8]
Jodi [suggested](http://lists.wikimedia.org/pipermail/wiki-research-l/2011-June/001587.html) that wikistream also monitor wikisource, wikiquote, etc. There will be some work to make sure that the IRC messages are similar I guess. It would be neat to be able to see images from the commons as they are uploaded.

**Comment by pizzamaker** [comment]
Agreed, this would be a neat idea. Also being able to watch "all wikisources", "all wikiquotes", etc
